sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -30,6 +30,7 @@ private[spark] object SQLConf {
   val DEFAULT_SIZE_IN_BYTES = "spark.sql.defaultSizeInBytes"
   val SHUFFLE_PARTITIONS = "spark.sql.shuffle.partitions"
   val CODEGEN_ENABLED = "spark.sql.codegen"
+  val UNSAFE_ENABLED = "spark.sql.unsafe.enabled"
```

Comment:
There was an option for choosing heap vs. offheap, but that option got moved into Spark core.  I think we use the `enabled` convention a bit more in core.  I don't have strong feelings on this one.